src/epub/text/chapter-2-1-8.xhtml

```diff
@@ -99,11 +99,11 @@
+				<p>I vainly pleaded my own total ignorance of music, and total want of taste in that direction. He only appealed to me again with a vehemence which set all further protest on my part at defiance. “The English and the Germans (he indignantly declared) were always reviling the Italians for their inability to cultivate the higher kinds of music. We were perpetually talking of our oratorios, and they were perpetually talking of their symphonies. Did we forget and did they forget his immortal friend and countryman, Rossini? What was <i epub:type="se:name.music.opera">Moses in Egypt</i> but a sublime oratorio, which was acted on the stage instead of being coldly sung in a concert-room? What was the overture to <i epub:type="se:name.music.opera">Guillaume Tell</i> but a symphony under another name? Had I heard <i epub:type="se:name.music.opera">Moses in Egypt</i>? Would I listen to this, and this, and this, and say if anything more sublimely sacred and grand had ever been composed by mortal man?”⁠—And without waiting for a word of assent or dissent on my part, looking me hard in the face all the time, he began thundering on the piano, and singing to it with loud and lofty enthusiasm⁠—only interrupting himself, at intervals, to announce to me fiercely the titles of the different pieces of music: “<span epub:type="se:name.music.song">Chorus of Egyptians in the Plague of Darkness</span>, Miss Halcombe!”⁠—“<span epub:type="se:name.music.song">Recitativo of Moses with the tables of the Law</span>.”⁠—“<span epub:type="se:name.music.song">Prayer of Israelites, at the passage of the Red Sea</span>. Aha! Aha! Is that sacred? is that sublime?” The piano trembled under his powerful hands, and the teacups on the table rattled, as his big bass voice thundered out the notes, and his heavy foot beat time on the floor.</p>
+				<p>There was something horrible⁠—something fierce and devilish⁠—in the outburst of his delight at his own singing and playing, and in the triumph with which he watched its effect upon me as I shrank nearer and nearer to the door. I was released at last, not by my own efforts, but by Sir Percival’s interposition. He opened the dining-room door, and called out angrily to know what “that infernal noise” meant. The Count instantly got up from the piano. “Ah! if Percival is coming,” he said, “harmony and melody are both at an end. The Muse of Music, Miss Halcombe, deserts us in dismay, and I, the fat old minstrel, exhale the rest of my enthusiasm in the open air!” He stalked out into the verandah, put his hands in his pockets, and resumed the “<span epub:type="se:name.music.song">recitativo of Moses</span>,” sotto voce, in the garden.</p>
```

Comment:
`Recitativo` should be capitalized

Reply:
Done! I made it as an editorial commit since it's not capitalized in the source scans.